### ISSUE TITLE
YAML rights attribute wrong parsed in contents.opf (ePub 2/3)

### ISSUE THREAD

**Issue by ousia** [jgm/pandoc#1844]
```
---
title:  'This is the title: it contains a colon'
rights: |
    These are the rights.

    In multiple paragraphs.
...
```

gives the following field in `contents.opf` (when parsing to ePub2/3):

``` html
<dc:rights>These are the rights.In multiple paragraphs.</dc:rights>
```

I must say that I don’t know whether `<dc:rights>` can have `<p>` as child element. (I haven’t find anything at http://dublincore.org/documents/.)

Would it be possible to add a space between paragraphs? Otherwise `<dc:rights>` is wrong.

**Comment by ousia** [comment]
Sorry for the delayed reply.

Tags are not allowed inside `<dc:rights>` (and, I guess, in any other `<dc>` element).

[epubcheck](https://github.com/IDPF/epubcheck) gives the following warnings for the following element (line 8 from `content.opf`):

```
    <dc:rights><p>These are the rights.</p></dc:rights>
```

with ePub version 2:

```
ERROR(RSC-005): book2.epub/content.opf(8,19): Error while parsing file 'element "p"
not allowed anywhere; expected the element end-tag or text'.
```

with ePub version 3:

```
ERROR(RSC-005): book3.epub/content.opf(8,19): Error while parsing file 'element "p"
not allowed anywhere; expected data'.
ERROR(RSC-005): book3.epub/content.opf(8,56): Error while parsing file 'element "dc:rights"
incomplete; expected data'.
```

**Comment by jgm** [comment]
Rendering with the plain text writer might be the best approach, then.

**Comment by jgm** [comment]
Closed by f1aaad9e86023dd90423681494dd24c48758c76f
